CMake/folly-config.h.cmake

```diff
@@ -26,6 +26,10 @@
 #endif
 #endif // FOLLY_MOBILE
 
+#if defined(__FreeBSD__)
+#define FOLLY_ASSUME_NO_TCMALLOC 1
```

Comment:
This is supposed to be an override.

Reply:
it was to make it handier for the FreeBSD package